constellation/src/storage/mod.rs

```diff
@@ -950,6 +966,7 @@ mod tests {
                     rkey: "asdf".into(),
                 },],
                 next: None,
+                total: 3,
```

Comment:
aha! version.0 differs from total in this test (as it should), so this test case should catch any future regressions for `/links` totals